packages/cf-component-radio/src/Radio.js

```diff
@@ -1,6 +1,86 @@
 import React from 'react';
-
 import PropTypes from 'prop-types';
+import { createComponent } from 'cf-style-container';
+
+const styles = ({ theme }) => ({
+  cursor: theme.cursor,
+  display: theme.display,
+  minHeight: theme.minHeight,
+  paddingLeft: theme.paddingLeft,
+  marginTop: theme.marginTop,
+
+  '&:first-child': {
+    marginTop: theme['&:first-child'].marginTop
+  },
+
+  '&:hover input': {
+    borderColor: theme['&:hover input'].borderColor
+  }
+});
+
+const Input = createComponent(
+  ({ theme, checked }) => ({
+    zIndex: 0,
+    border: '1px solid #bdbdbd',
```

Comment:
This color: `#bdbdbd`.

storm:        `#808285`
hail:            `#BCBEC0`
smoke:        `#e0e0e0`

Hail feels like the closest one, I'll go with that.